configure

```diff
@@ -53,9 +53,29 @@ Options:
   --spidermonkey-version VSN  specify the version of SpiderMonkey to use (defaults to $SM_VSN)
   --skip-deps                 do not update erlang dependencies
   --rebar=PATH                use rebar by specified path (version >=2.6.0 && <3.0 required)
+  --tls                       enable TLS distribution and generate a certfile
```

Comment:
Let's rename the `--tls` option itself to `--generate-tls-dev-cert` so it's more descriptive. And we'd note that it just generates a cert but doesn't actually enable TLS distribution. Enabling would happen in vm.args file.

Reply:
Changed.